common/app/html/HtmlPage.scala

```diff
@@ -42,8 +42,8 @@ object HtmlPageHelpers {
     )
   }
 
-  def FaciaCSSFile: String = if (PillarCards.isSwitchedOn) "facia.pc" else "facia"
-  def ContentCSSFile: String = if (PillarCards.isSwitchedOn) "content.pc" else "content"
-  def RichLinksCSSFile: String = if (PillarCards.isSwitchedOn) "rich-links.pc" else "rich-links"
+  def FaciaCSSFile: String = if (PillarCards.isSwitchedOn || mvt.PillarCards.isParticipating) "facia.pc" else "facia"
```

Comment:
Shouldn't it be a `&&`?

Reply:
Which is why you can simply use `isParticipating` which I believe check if the switch is ON

Reply:
@TBonnin you are correct. This should probably not be a switch now, but a serverside AB test with no audience except opt in. 

Best to look at how the New Header switch works here: 
https://github.com/guardian/fastly-edge-cache/blob/master/theguardiancom/src/main/resources/varnish21/ab-tests.vcl#L30

Reply:
does that mean to release it, you have to make a change to fastly edge cache, rather than enable/disable the swtich?

Reply:
Yes. But I imagine you want a slow roll out just like the polyfill so you would want the ability to control how much of the audience sees this change.

Reply:
But you can have the audience at 100% and turn the switch off, and then turn it on for the day.

Reply:
perfect - thanks both for guiding me through this!